operations/power_management/Recover_from_a_Liquid_Cooled_Cabinet_EPO_Event.md

```diff
@@ -13,15 +13,27 @@ If a Cray EX liquid-cooled cabinet or cooling group experiences an EPO event, th
 2. From `ncn-m001`, check the status of the chassis.
```

Comment:
Reference to CAPMC in intro paragraph above

Reply:
Ref OK, having an example bad.